### PR TITLE
Ensure the fork tests are correctly running the upcoming release

### PR DESCRIPTION
**Pull request by justinjmoses**
A number of problems are fixed in this PR:

1. The `releases.json` had two entries with `released: false` and *Mirfak* should have been marked as `ovm: false` (the latter prevented the CI `fork-tests` from properly deploying the changed contracts
2. SIP-150 had been merged into `develop` and was causing `Cached synth sum exceeds total debt` errors (which is why 150 was rejected in the first place). It had to be reverted along with the changed test.
3. There was a bug in `Deployer` when it was migrated to `ethers` that `deployed` was showing as `truthy` and hence the deploy script was showing that every contract was redeployed when they weren't.

### CODE REVIEW

File: publish/releases.json [hunk 1]
```diff
@@ -278,7 +278,7 @@
 	},
 	{
 		"name": "Dubhe (Optimism)",
-		"released": false,
```
Comment (justinjmoses) [inline publish/releases.json:281]: This was indeed released. I think the `prepublishOnly` script should amend the last entry here to `true`. Thoughts @ajsantander ?

File: publish/releases.json [hunk 2]
```diff
@@ -292,17 +292,16 @@
 		"released": false,
 		"version": {
 			"major": 2,
-			"minor": 47
```
Comment (justinjmoses) [inline publish/releases.json:295]: Mirfak should increment this number

File: publish/releases.json [hunk 3]
```diff
@@ -292,17 +292,16 @@
...
+			"minor": 48
 		},
-		"ovm": true,
```
Comment (justinjmoses) [inline publish/releases.json:297]: This was incorrect

File: publish/releases.json [hunk 4]
```diff
@@ -292,17 +292,16 @@
...
 		"sources": [
-			"Synthetix",
```
Comment (justinjmoses) [inline publish/releases.json:299]: sort ascending for easy reading

File: publish/releases.json [hunk 5]
```diff
@@ -292,17 +292,16 @@
...
 			"DebtCache",
-			"SynthsUSD",
-			"SynthsETH"
+			"FeePool",
+			"Issuer",
+			"Synthetix",
+			"SynthsETH",
+			"SynthsUSD"
 		],
-		"sips": [142]
```
Comment (justinjmoses) [inline publish/releases.json:306]: adding 145 (already merged into DebtCache a while back and thus will be released with 142, and 170 for inflation diversion to L2)

File: publish/src/Deployer.js [hunk 6]
```diff
@@ -409,7 +409,7 @@ class Deployer {
 		await this._updateResults({
 			name,
 			source: deployedContract.source,
-			deployed: deployedContract.deployed,
```
Comment (justinjmoses) [inline publish/src/Deployer.js:412]: `deployed` is a function on an ethers contract and hence it was coercing to a truthy value.

File: publish/src/commands/deploy/system-and-parameter-check.js [hunk 7]
```diff
@@ -215,20 +215,20 @@ module.exports = async ({
 			: yellow('⚠ NO'),
 	});
 
+	console.log(
```
Comment (justinjmoses) [inline publish/src/commands/deploy/system-and-parameter-check.js:218]: Moving this above the confirm so even when run with `--yes` we can see which contracts the deploy script is set to update

**Review by leomassazza — APPROVED** [review]
LGTM

**Pull request merged**